apps/passport-client/public/index.hbs

```diff
@@ -29,7 +29,14 @@
     <script defer src="/js/index.js"></script>
   </head>
   <body>
-    <div id="root" />
+    <div id="root">
```

Comment:
nit: add a comment explaining that this shows the RippleLoader in the background